test/integration/application_controller_test.rb

```diff
@@ -155,7 +155,7 @@ def with_test_routes
         get '/client_caching'
       end
 
-      assert_equal 'no-cache, no-store', response.headers['Cache-Control']
+      assert_equal 'no-store', response.headers['Cache-Control']
```

Comment:
So, Rails 6.1 cuts out the `no-cache` value from the header, due to [this normalization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control). According to [the docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) `no-store` is "stronger" than `no-cache`, which indeed makes `no-cache` redundant in presence of `no-store`, so I changed the headers we set in `ApplicationController#disable_client_cache`.

Reply:
@akostadinov as @jlledom has pointed, this is the explanation for the no-cache change :point_up: 

(I've updated the first link that was wrong)